tests/ui/test_ui_tools.py

```diff
@@ -1150,6 +1157,9 @@ def test_keypress_GO_BACK(self, right_col_view, mocker, key, widget_size):
 
         right_col_view.set_body.assert_called_once_with(right_col_view.body)
         right_col_view.set_focus.assert_called_once_with("body")
+        right_col_view.body.log.set_focus.assert_called_once_with(
```

Comment:
This doesn't check whether the focus before and after the search match. Currently, this passes even when I remove the `self.focus_index_before_search = self.body.log.get_focus()[1]` from the other `if` block. (This was supposed to go with my last review.)

Reply:
But that would mean the other test fails right? The current tests for userlist are checking only what each function is doing. I'm not able to test this like the previous 2 for some reason

Reply:
I think we can come back to this after the Panel refactor when the tests also become simpler?

Reply:
Generally, tests shouldn't rely on other tests to fail. Let's revisit it post the refactor. :+1: